### DISCUSSION TITLE
querying heavy and light chain separately or together?

### DISCUSSION THREAD

**Discussion by avilella** [brianhie/efficient-evolution#10] [Q&A]
Dears,

I've just found out about this tool, successfully installed it in an Ubuntu Linux computer with an NVidia GPU card with 16Gb VRAM, and ran the prediction script successfully (recommend.py).

I would like to know what's the recommended way of predicting the mutations for the heavy and the light chain (Fv) of a mAb. Should I do it separately for the heavy and the light?

To predict the pdb structure, we use heavy+{GGGGS}x4+light as a single chain input, with the linker being long enough not to interfere with the prediction.

Would that single chain be a good input for the prediction? Or is it better to predict separately the heavy and the light chains?

Thanks in advance

**Comment by brianhie** [comment]
We have just been providing the VH and VL sequences separately, e.g., https://github.com/brianhie/efficient-evolution/blob/5be1d890855bb41720655bcf2b5f6ab9ddeeba76/bin/medi8852.py#L30

However, you could try concatenating the sequences (and adding a linker).
